### ISSUE TITLE
hamster-cli with export fails

### ISSUE THREAD

**Issue by mabgh** [projecthamster/hamster#373]
```python
hamster export html 2018-12-01 2018-12-31
  File "/usr/bin/hamster", line 391, in <module>
    getattr(hamster_client, command)(*args)
  File "/usr/bin/hamster", line 218, in export
    writer = reports.simple(facts, start_time, end_time, export_format)
  File "/usr/lib/python3.6/site-packages/hamster/reports.py", line 60, in simple
    writer.write_report(facts)
  File "/usr/lib/python3.6/site-packages/hamster/reports.py", line 101, in write_report
    self._finish(facts)
  File "/usr/lib/python3.6/site-packages/hamster/reports.py", line 339, in _finish
    self.file.write(Template(self.main_template).safe_substitute(data))
  File "/usr/lib64/python3.6/codecs.py", line 377, in write
    self.stream.write(data)
TypeError: string argument expected, got 'bytes'
```
with   lines 339 in `reports.py` as
```python
    data = Template(self.main_template).safe_substitute(data)
    print(data) 
```
data is class str and a correct html is printed to stdout

**Comment by ederag** [comment]
Thanks for the report.
Which hamster version are you using ?

**Comment by mabgh** [comment]
Am Sonntag, 6. Januar 2019, 22:01:18 CET schrieb ederag:
> Thanks for the report.
> Which hamster version are you using ?

v2.1.1 - python3 migration (zip) from releases [1]

--------
[1] https://github.com/projecthamster/hamster/releases/tag/
v2.1.1

**Comment by ederag** [comment]
Thanks, confirmed. I'll look into that.

**Comment by ederag** [comment]
Got it. The issue was two-fold. The second part is
 https://github.com/projecthamster/hamster/blob/7d2a8b46c1de5f57eb6bcee3174afa67777bbf8b/src/hamster/reports.py#L78

What output is expected from the command-line ?
(just print the html to standard output, for redirection ?)

**Comment by ederag** [comment]
Should be fixed by PR #375.
